File: flutter_kakao/lib/Model/kakao_user_model.dart


class KakaoUserModel{
  String _nickName;
  String _email;
  String _gender;
  String _birthday;
  String _birthyear;
  String _phoneNumber;


  KakaoUserModel(String _nickName, String _email, String _gender, String _birthday,
      String _birthyear, String _phoneNumber){
    this._nickName = _nickName;
    this._email = _email;
    this._gender = _gender;
    this._birthday = _birthday;
    this._birthyear = _birthyear;
    this._phoneNumber = _phoneNumber;
  }

  String get nickName => _nickName;

  String get email => _email;

  String get phoneNumber => _phoneNumber;

  String get birthyear => _birthyear;

  String get birthday => _birthday;

  String get gender => _gender;
}

File: kakao_android/app/src/main/java/com/example/kakao_android/KakaoUserModel.java
package com.example.kakao_android;

import java.io.Serializable;

public class KakaoUserModel implements Serializable {
    private String nickname;
    private String email;
    private String gender;
    private String birthday;

    public KakaoUserModel(String nickname, String email, String gender, String birthday) {
        this.nickname = nickname;
        this.email = email;
        this.gender = gender;
        this.birthday = birthday;
    }

    public String getNickname() {
        return nickname;
    }

    public void setNickname(String nickname) {
        this.nickname = nickname;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public String getBirthday() {
        return birthday;
    }

    public void setBirthday(String birthday) {
        this.birthday = birthday;
    }
}
